plugins/graphql-java-codegen-gradle-plugin/graphql-codegen-gradle-plugin/src/main/java/io/github/kobylynskyi/graphql/codegen/gradle/GraphqlCodegenGradleTask.java

```diff
@@ -37,6 +35,8 @@
     private Boolean generateEqualsAndHashCode = false;
     private Boolean generateToString = false;
     private Boolean generateAsyncApi = false;
+    private Boolean generateParameterizedFieldsResolvers = true;
+    private Set<String> fieldsResolvers = new HashSet<>();
```

Comment:
Shouldn't this be `fieldsWithResolvers` instead? I think we're listing fields here, not field resolvers. What do you think?

Reply:
Renamed the field as you suggested.